sdk/forms/README.md

```diff
@@ -10,130 +10,276 @@
+
+1. The Vault can be configured to allow communication only from the allowed origin of the form.
+2. The form iframe is served with strict CSP (Content Security Policy) headers, making it a sandbox that prevents PII from leaking out, while allowing exclusive communication with the Vault.
+
+This architecture ensures that even if the user frontend is compromised (e.g., via a malicious package or XSS attack), the attacker cannot access the iframe content or manipulate the communication with the Vault.
+
+## Setup
+
+To use Piiano Forms, you need to have a Piiano Vault instance set up.
+
+If you don’t have one, you can [sign up for a free sandbox vault](https://app.piiano.io/register) to get started.
+
+Once you have your Vault set up, you can include the `@piiano/forms` package in your project with a package manager or directly from our CDN.
+
+### npm
+
+```bash
+npm install @piiano/forms
 ```
 
-The form is pretty standard, but we need to add a few things to make it work with Piiano Forms.
+### yarn
 
-First, we need to include the Piiano Forms SDK using one of the following methods:
-- Include the SDK from Piiano CDN by including the Piiano Forms SDK in the head of the page:
-  ```html
-  <script src="https://cdn.piiano.com/pvault-forms-lib-v1.0.24.js"></script>
-  ```
-- Install the SDK using npm:
-  ```bash
-  npm install @piiano/forms
-- Install the SDK using yarn:
-  ```bash
-  yarn add @piiano/forms
-  ```
+```bash
+yarn add @piiano/forms
+```
 
-Next, add a `data-piiano-proxy-options` attribute to the form and set it to a JSON configuration object:
+### pnpm
+
+```bash
+pnpm add @piiano/forms
+```
+
+### CDN
 
 ```html
-<form
-  action="/cards"
-  method="post"
-  data-piiano-proxy-options='{
-        "collection":"credit_cards",
-        "hijack":true,
-        "vaultURL":"https://xxxxxxxxxx.us-east-2.awsapprunner.com"
-      }'
->
-  <h1>Add payment details</h1>
-  <label for="card_number">Card Number</label>
-  <input type="text" id="card_number" name="card_number" />
-  <label for="card_holder">Card Holder</label>
-  <input type="text" id="card_holder" name="card_holder" />
-  <label for="card_expiry">Card Expiry</label>
-  <input type="text" id="card_expiry" name="card_expiry" />
-  <label for="card_cvv">Card CVV</label>
-  <input type="text" id="card_cvv" name="card_cvv" />
-  <button type="submit">Add card</button>
-</form>
+<script src="https://cdn.piiano.com/pvault-forms-lib-v1.0.43.js"></script>
+```
+
+> **Note**
+> 
+> The CDN version of the library is available in few versions:
+> - `pvault-forms-lib-latest.js`: The latest version of the library.
+> - `pvault-forms-lib-v1.js`: The latest version with the v1 major version.
+> - `pvault-forms-lib-v1.0.js`: The latest version in the v1.0 major & minor version.
+> - `pvault-forms-lib-v1.0.43.js`: Get an exact version of the library by specifying the version number.
+> 
+> It is recommended to use the exact version of the library in production to avoid breaking changes.
```

Comment:
```suggestion
> It is recommended to use the specific version of the library in production and not the latest version to avoid breaking changes.
```